plugins/package-managers/gradle-plugin/src/main/kotlin/GradleModelExtensions.kt

```diff
@@ -44,7 +44,10 @@ internal fun Configuration.isRelevant(): Boolean {
     val isDeprecatedConfiguration = GradleVersion.current() >= GradleVersion.version("6.0")
             && this is DeprecatableConfiguration && resolutionAlternatives != null
 
-    return canBeResolved && !isDeprecatedConfiguration
+    // Do not try to resolve dependencies metadata configurations as there often cause failures with Gradle itself.
+    val isDependenciesMetadata = name.endsWith("DependenciesMetadata")
+
```

Comment:
could we include a reference in the code comment to what this "*DependenciesMetadata" is about, and maybe also why it is ok to filter it out?

Reply:
It's actually quite hard to find information about this in other places than those already given in the commit message (of the original commit that introduced the check for Android). But I've added those links here, too, for completeness.

Reply:
Thanks!